compiler/rustc_session/src/options.rs

```diff
@@ -1072,6 +1086,11 @@ options! {DebuggingOptions, DebuggingSetter, basic_debugging_options,
         "hash algorithm of source files in debug info (`md5`, or `sha1`)"),
     strip: Strip = (Strip::None, parse_strip, [UNTRACKED],
         "tell the linker which information to strip (`none` (default), `debuginfo` or `symbols`)"),
+    split_dwarf: SplitDwarfKind = (SplitDwarfKind::None, parse_split_dwarf_kind, [UNTRACKED],
+        "enable generation of split dwarf"),
+    split_dwarf_inlining: bool = (true, parse_bool, [UNTRACKED],
```

Comment:
bikeshed colour nit: verb this into `inline_split_dwarf`

Reply:
This is split dwarf info with inlining info I think.

Reply:
My reading of the LLVM code and the description of this flag itself suggests that this flag controls whether minimal debug info should also be generated within CUs.

Reply:
Yes, but that minimal debug info is I think debug info necessary for handling function inlining in backtraces.

Reply:
I think the current wording makes the most sense, but I don't feel strongly about it.